cypress/e2e/forgotPassword.cy.js

```diff
@@ -0,0 +1,28 @@
+import forgotPasswordPage from '../support/page_objects/forgotPasswordPage';
+import * as page from '../fixtures/URLs.json';
+
+describe('ForgotPassword page spec', () => {
+  it('should show a validation error when the email field is left blank', () => {
+    cy.visit(page.forgotPassword);
+    forgotPasswordPage.confirmResetButton().click();
+    cy.get('input[name="email"]').then(($input) => {
+      expect($input[0].validationMessage).to.eq('Please fill in this field.');
```

Comment:
This test is failing because the actual message says "fill out" instead of "fill in". But we can just make it a RegEx as well and make the test sturdier

```js
expect($input[0].validationMessage).to.eq(/please fill/gi);
```

Reply:
You are right, thank you. Just small change, instead of .eq I use .match() to compare the string against a regular expression. The .eq() compares strict equality and doesn't work with regular expressions.